lib/ansible/modules/user.py

```diff
@@ -433,6 +445,16 @@
   returned: When user exists
   type: int
   sample: 20
+subuid:
+  description: UID range allocated for use by newuidmap
+  returned: When subuid range has been allocated
+  type: str
+  sample: 100000-165535
```

Comment:
```suggestion
  sample: 100000-165535
  version_added: 2.12
```
(Same below.)

Also, why not return this in a way that can be processed without parsing it first? I.e. provide start and end (or start, end and count) as separate values, or a dictionary with fitting keys?